### ISSUE TITLE
cozy-dev proxify doesn't work

### ISSUE THREAD

**Issue by jacquarg** [cozy/cozy-dev#96]
Used as described https://dev.cozy.io/clientsideapp.html#make-a-client-side-quot-hello-world-application-quot , in a client side app (mesinfosdataplayground)

```
toto@monpc$ cozy-dev proxify http://localhost:9104
? Cozy password for http://localhost:9104: *******
info - profixy	 | SHELL FOR 
readline.js:930
            throw err;
            ^

TypeError: Cannot read property 'url' of undefined
    at Object.module.exports.start (/home/jacquarg/opt/node/lib/node_modules/cozy-dev/lib/static-proxy.js:169:29)
    at /home/jacquarg/opt/node/lib/node_modules/cozy-dev/lib/main.js:195:26
    at PromptUI.completed (/home/jacquarg/opt/node/lib/node_modules/cozy-dev/lib/helpers.js:87:16)
    at PromptUI.onCompletion (/home/jacquarg/opt/node/lib/node_modules/cozy-dev/node_modules/inquirer/lib/ui/prompt.js:69:10)
    at AnonymousObserver.Rx.AnonymousObserver.AnonymousObserver.completed (/home/jacquarg/opt/node/lib/node_modules/cozy-dev/node_modules/inquirer/node_modules/rx/dist/rx.js:1793:12)
    at AnonymousObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (/home/jacquarg/opt/node/lib/node_modules/cozy-dev/node_modules/inquirer/node_modules/rx/dist/rx.js:1730:14)
    at AnonymousObserver.tryCatcher (/home/jacquarg/opt/node/lib/node_modules/cozy-dev/node_modules/inquirer/node_modules/rx/dist/rx.js:567:29)
    at AutoDetachObserverPrototype.completed (/home/jacquarg/opt/node/lib/node_modules/cozy-dev/node_modules/inquirer/node_modules/rx/dist/rx.js:5288:56)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (/home/jacquarg/opt/node/lib/node_modules/cozy-dev/node_modules/inquirer/node_modules/rx/dist/rx.js:1730:14)
    at InnerObserver.onCompleted (/home/jacquarg/opt/node/lib/node_modules/cozy-dev/node_modules/inquirer/node_modules/rx/dist/rx.js:3425:65)
```

**Comment by aenario** [comment]
@jacquarg temporarily, you need a package.json `repository.url`, fix is on master but still need publishing

**Comment by jacquarg** [comment]
Ok good to know ! 
In deed, it works with the cozy-dev from the master

**Comment by aenario** [comment]
released v1.0.28

**Comment by jacquarg** [comment]
Thank's for that. But I have another problem now (with the 1.0.28) : 

```
cozy-dev proxify http://localhost:9104
? Cozy password for http://localhost:9104: ********
info - profixy	 | SHELL FOR 
readline.js:930
            throw err;
            ^

Error: Cannot find module 'npmconf'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/jacquarg/opt/node/lib/node_modules/cozy-dev/misc/tunnel.js:7:15)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
```

May be it's because npmconf is in devDependencies (https://github.com/cozy/cozy-dev/blob/master/package.json#L56) ?
(or may be my npm setup is not clean).

**Comment by nono** [comment]
I think it has been fixed in master but not yet published

**Comment by doubleface** [comment]
I confirm this works with the master

**Comment by nono** [comment]
It was published as v1.0.28 two days ago.
